spinalcordtoolbox/scripts/sct_run_batch.py

```diff
@@ -437,14 +442,14 @@ def reset_streams():
         # Only consider folders
         if os.path.isdir(os.path.join(path_data, isub)):
             session_dirs = [f for f in os.listdir(os.path.join(path_data, isub)) if f.startswith('ses-')]
-            if not session_dirs:
-                # There is no session folder, so we consider only sub- directory: sub-XX
-                subject_dirs.append(isub)
-            else:
-                # There is a session folder, so we concatenate: sub-XX/ses-YY
+            if session_dirs and arguments.ignore_ses is not True:
```

Comment:
Thank you for this! I think I may have reflexively wrote `is not True` because it sounded better in terms of spoken English. (Even though, as you've pointed out, what sounds good in English isn't necessarily functionally correct. :stuck_out_tongue_winking_eye:)